meerk40t/gui/wxmribbon.py

```diff
@@ -214,7 +214,7 @@ def open_simulator(v=None):
         )
         toolbar.Bind(
             RB.EVT_RIBBONBUTTONBAR_CLICKED,
-            lambda e: self.context(".dialog_save\n"),
+            lambda e: self.context(".on_click_save\n"),
```

Comment:
I don't think this will work as there isn't a console command registered for `on_click_save`.

Reply:
hmm it's definitely working for me, I thought it just keyed off of the method name

Reply:
Yup.